templates/trivy-step.tmpl

```diff
@@ -1,12 +1,33 @@
 steps:
-- command: ls
-  plugins:
-    - equinixmetal-buildkite/trivy#{{ .TrivyPlugin }}:
-        severity: {{ join .Severity }}
-        ignore-unfixed: {{ .IgnoreUnfixed }}
-        security-checks: {{ join .SecurityChecks }}
-        skip-files: '{{ .SkipFiles }}'
-- label: ":sparkles: SHELL CHECK"
-  plugins:
-    - shellcheck#{{ .ShellPlugin }}:
-        files: script.sh
+  - command: ls
+    plugins:
+{{- if .TrivyPluginEnabled }}
+      - equinixmetal-buildkite/trivy#{{ .TPConfig.TrivyVersion }}:
+          {{- if .TPConfig.ExitCode}}
+          exit-code: {{ .TPConfig.ExitCode }}
+          {{- end }}
+          {{- if .TPConfig.Timeout}}
+          timeout : {{ .TPConfig.Timeout }}
+          {{- end }}
+          {{- if .TPConfig.Severity}}
+          severity: {{ .TPConfig.Severity }}
+          {{- end }}
+          {{- if .TPConfig.IgnoreUnfixed}}
+          ignore-unfixed: {{ .TPConfig.IgnoreUnfixed }}
+          {{- end }}
```

Comment:
Is this something we want users to configure? I would think we always want this as true?

Reply:
completed